create-repo.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
```

Comment:
Since this has a useful shebang line, it'd be good to mark it as executable as well.

Reply:
Done!